src/components/views/Features/Features.module.css

```
@@ -1,113 +1,112 @@
+    margin-bottom: 1.25rem;
+}
 
-        .title {
-            font-size: 2rem;
-            display: inline-block;
-            color: var(--fg);
-            font-weight: 800;
-            vertical-align: bottom;
-        }
+.title {
+    font-family: var(--font-instrument-serif), "Georgia", serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Georgia" (font-family-name-quotes)

(font-family-name-quotes)